bpfctl/src/main.rs

```diff
@@ -105,9 +112,9 @@ struct LoadFileArgs {
     /// Only used when multiple eBPF programs need to share a map. If a map is being
     /// shared with another eBPF program, the eBPF program that created the map can not
     /// be unloaded until all eBPF programs referencing the map are unloaded.
-    /// Example: --map-owner-uuid 989958a5-b47b-47a5-8b4c-b5962292437d
+    /// Example: --map-owner-id 63178
```

Comment:
Line 111 and 146:
```
-    /// Optional: UUID of loaded eBPF program this eBPF program will share a map with.
-    /// Only used when multiple eBPF programs need to share a map. If a map is being
-    /// shared with another eBPF program, the eBPF program that created the map can not
-    /// be unloaded until all eBPF programs referencing the map are unloaded.

+    /// Optional: Id of loaded eBPF program this eBPF program will share a map with.
+    /// Only used when multiple eBPF programs need to share a map.
```

Reply:
Done

Reply:
Nit: This was existing, but since you were there, this sentence needs to be removed:

```
If a map is being shared with another eBPF program, the eBPF program that created the map can not be unloaded until all eBPF programs referencing the map are unloaded.
```